operations/reservations.md

```diff
@@ -35,13 +35,13 @@ Returns all reservations from the specified interval according to the time filte
 
 #### Reservation time filter
 
-* `Colliding` - reservation interval collides with the interval.
-* `Created` - reservation created within the interval.
-* `Updated` - reservation last update date and time within the interval.
-* `Start`- reservation start \(= arrival\) within the interval.
-* `End` - reservation end \(= departure\) within the interval.
-* `Overlapping` - reservation interval contains the specified interval.
-* `Canceled` - reservation cancelled within the specified interval.
+* `Colliding` - reservation intervals which collide with the specified interval.
+* `Created` - reservations created within the specified interval.
+* `Updated` - reservations updated within the specified interval.
+* `Start`- reservations starting \(= arriving\) within the specified interval.
+* `End` - reservations finishing \(= departing\) within the specified interval.
+* `Overlapping` - reservation intervals which are contained within the specified interval.
```

Comment:
does not make sense, should be "reservations whose intervals contain the specified interval"